.storybook/recipes/MenuWithIconButton.stories.tsx

```diff
@@ -20,11 +20,11 @@ export default {
       defaultValue: 'dots-vertical',
     },
   },
-} as Meta<MenuProps>;
```

Comment:
Point of story is to have different icons as menu buttons. This previously should have caused type error and now does, changed to iconName arg only